.travis.yml

```diff
@@ -108,9 +108,9 @@ script:
     - /tmp/julia/bin/julia --precompiled=no -e 'true' &&
         /tmp/julia/bin/julia-debug --precompiled=no -e 'true'
     - /tmp/julia/bin/julia -e 'versioninfo()'
-    - export JULIA_CPU_CORES=2 && export JULIA_TEST_MAXRSS_MB=600 && cd /tmp/julia/share/julia/test &&
```

Comment:
why not do ```v`cut -d. -f1-2 < julia2/VERSION` ``` for this? would rather leave the cd and direct use of runtests.jl for now